src/SearchByTitleForm/SearchByTitleForm.js

```diff
@@ -1,30 +1,34 @@
 import React, { Component } from 'react';
 import '../Navbar/Navbar.css';
-import { Link } from 'react-router-dom'
-import { Swtich } from 'react-router-dom'
+import { Redirect, NavLink } from 'react-router-dom'
 
 class SearchByNameForm extends Component {
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
         this.state = {
             movieTitle: '',
-            id: ''
+            id: this.props.id || ''
```

Comment:
here is where the prop ends up, I am not sure if this is doing anything at all so yeah